.github/workflows/vercel-preview-shortener.yaml

```
@@ -1,74 +1,45 @@
-env:
-  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
-  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_SHORTENER_PROJECT_ID }}
-  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
 on:
-  push:
-    branches-ignore:
-      - production
-  workflow_dispatch:
+  workflow_run:
+    workflows: [ "Detect Changes for Apps" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)